### DISCUSSION TITLE
Please restore longer model version names

### DISCUSSION THREAD

**Discussion by MoonRide303** [civitai/civitai#446] [Ideas]
About 2 weeks ago I was able to name my model version properly, using full name, that is "MoonRide Mix 9". I released "MoonRide Mix 10" yesterday, but I was unable to enter full version name "MoonRide Mix 10" and I had to shorten it to "Mix 10", because currently longer model version names are not allowed.

Please allow longer model version names, like 20-30 characters - it should be kinda equivalent of git tag names, not just few characters.

**Comment by MoonRide303** [comment]
I've just tried it today (when website style changed to dark colors), and it seems to be working, now.

**Comment by JustMaier** [comment]
Hey, yes, we just bumped the limit up to 25 characters from 10. That said, we intentionally lowered the limit to encourage people to not repeat the name of the model in the version. So in your case, if your model was "MoonRide" then "Mix 10" is actually the kind of version name we're trying to get people to lean towards.

**Reply by MoonRide303** [reply]
@JustMaier I get it, but even in my private repo I use longer and a bit more descriptive version names ("MoonRide Mix 10", not "Mix 10") - and I think it should be allowed for model authors to pick their own naming patterns, with which they feel most comfortable. Thank you for raising this limit - 25 sounds reasonable, and should be enough for everyone.
